utils/build-script

```diff
@@ -89,6 +88,25 @@ def print_xcodebuild_versions(file=sys.stdout):
     file.flush()
 
 
+def tar(source, destination):
+    """
+    Create a gzip archive of the file at 'source' at the given
+    'destination' path.
+    """
+    # We do not use `tarfile` here because:
+    #  - We wish to support LZMA2 compression while also supporting Python 2.7.
```

Comment:
Pretty sure we've never used this for LZMA2 compression, thus we should just use the builtin `tarfile` library. I'll do that in a follow-up PR.

Reply:
The builtin `tarfile` module supports lzma as well I *think*

Reply:
Only for Python 3.2 (or 3.3) and beyond. Python 2.7 supports gzip however, which appears to be all we use for the time being.